### PR TITLE
Added USA Banner

### PR DESCRIPTION
**Pull request by lgarceau768**
## Change Details
Added in USA Banner into the DPC Portal
Source code was pulled originally from here [https://design.cms.gov/components/usa-banner?theme=core](https://design.cms.gov/components/usa-banner?theme=core)
- added in the `dpc-web`

No new dependencies or security implications. This is pretty much a direct copy and paste from the [BCDA PR](https://github.com/CMSgov/bcda-static-site/pull/152)

### CODE REVIEW

**Review by mftcodes — APPROVED** [review]

**Review by TiannaW24 — CHANGES_REQUESTED** [review]
This is on hold until I can find out if this is needed for the web-apps

File: dpc-web/app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -66,3 +66,163 @@ aria-controls="topnav-wrap" aria-label="Close menu">
     </ul>
   </div>
 </nav>
+<section class="ds-c-usa-banner" aria-label="Official government website">
```
Comment (TiannaW24) [inline dpc-web/app/views/shared/_navbar.html.erb:69]: Can we move this code to its own module? Similar to what you did here: https://github.com/CMSgov/dpc-static-site/pull/72/files

Comment (TiannaW24) [inline dpc-web/app/views/shared/_navbar.html.erb:69]: This code is also in the wrong spot. The USA Banner should be above the entire nav. 
Screen Shot 2022-08-31 at 10 14 28 AM

**Review by TiannaW24 — CHANGES_REQUESTED** [review]
A couple of small changes regarding nav + code placement

File: dpc-admin/app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -44,3 +44,163 @@ aria-controls="topnav-wrap" aria-label="Close menu">
   </div>
   <% end %>
 </nav>
+<section class="ds-c-usa-banner" aria-label="Official government website">
```
Comment (TiannaW24) [inline dpc-admin/app/views/shared/_navbar.html.erb:47]: We don't need to add the banner to the admin app, since it's not customer-facing.

**Review by TiannaW24 — APPROVED** [review]
Looks good!

**Pull request merged**